<!DOCTYPE HTML>
<title>Canvas test: context.arguments.missing</title>
<!--[if IE]><script src="../excanvas/excanvas.js"></script><![endif]-->
<script src="../tests.js"></script>
<link rel="stylesheet" href="../tests.css">
<link rel="prev" href="context.casesensitive.html" title="context.casesensitive">
<link rel="next" href="context.arguments.extra.html" title="context.arguments.extra">
<body class="show_output">
<p>
 <a href="context.casesensitive.html" accesskey="p" title="[p] context.casesensitive">&lt;</a>
 <a href="index.html">[index]</a>
 <a href="context.arguments.extra.html" accesskey="n" title="[n] context.arguments.extra">&gt;</a>
<h1><a href="index.context.html">context</a>.<a href="index.context.arguments.html">arguments</a>.missing</h1>
<p class="desc"></p>
<div class="refs">References:
<ul>

</ul>
</div>

<p class="output">Actual output:</p>
<canvas id="c" class="output" width="100" height="50"><p class="fallback">FAIL (fallback content)</p></canvas>

<ul id="d"></ul>
<script>
_addTest(function(canvas, ctx) {

try { var _thrown = false;
  canvas.getContext();
} catch (e) { if (e.code != DOMException.NOT_SUPPORTED_ERR) _fail("Failed assertion: expected exception of type NOT_SUPPORTED_ERR, got: "+e.message); _thrown = true; } finally { _assert(_thrown, "should throw exception of type NOT_SUPPORTED_ERR: canvas.getContext()"); }


});
</script>

